src/renderer/components/atoms/text/helper/buildStylesForDocument.ts

```diff
@@ -0,0 +1,85 @@
+import type { TextUsage } from '../types';
+
+const buildStylesForDocument = (as: TextUsage): string => {
+  let styles = ['font-serif'];
```

Comment:
`styles` can be a `const` and the `concat` operations below would still be valid
```suggestion
  const styles = ['font-serif'];
```